spec/models/user_spec.rb

```diff
@@ -87,4 +83,42 @@
       expect(user.token).to be_nil
     end
   end
+
+  describe "#connect_with_invitors" do
+    context "invited" do
+      let!(:user) { Fabricate(:user) }
+      let!(:invitor) { Fabricate(:user) }
+      let!(:invite) { Fabricate(:invite, user: invitor, email: user.email) }
+
+      it "creates mutual followerships" do
+        user.connect_with_invitors
+        expect(Followership.first.leader).to eq(invitor)
+        expect(Followership.first.follower).to eq(user)
+        expect(Followership.last.leader).to eq(user)
+        expect(Followership.last.follower).to eq(invitor)
+      end
+
+      it "creates one set of followerships from same invitor" do
```

Comment:
Your spec does verify that there are two (I assume this is what you mean by "a set") followerships, but it doesn t verify the second part...that they are from the same inviter. Make sure that your spec descriptions and implementations always match.